BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -93,9 +92,6 @@ def create_stochastic_schedules(args:)
     # initialize a random number generator
     prng = Random.new(get_random_seed)
 
-    # load the schedule configuration file
-    schedule_config = JSON.parse(File.read(args[:resources_path] + '/schedules_config.json'))
```

Comment:
We no longer need to require json at the top of the file, right?

Reply:
Good catch.